client/gutenberg/extensions/plugins/atavist/plugin.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
Ouch, a PHP plugin in Calypso. Doesn't seem to be the right place for this - if this was going in Jetpack, I'd suggest  an accompanying PR in Jetpack instead. Then we wouldn't be needing any of it if we use the Jetpack blocks preset.

Reply:
This feels still an open question.

If the Atavist block should always depend on Jetpack, then it makes sense to move PHP to Jetpack as well. If it needs to work with both Jetpack and Atavist somewhere else without Jetpack, it gets more complicated.

cc @dmsnell

Reply:
Discussion here: p1537557809000100-slack-jetpack-gutenberg

Reply:
yeah 🤷‍♂️ we're playing around with it. there's no reason PHP can't live in Calypso's repo. I think it's worthwhile exploring but I don't have an answer.